ceno_zkvm/src/scheme.rs

```diff
@@ -208,10 +213,10 @@ impl<E: ExtensionField, PCS: PolynomialCommitmentScheme<E> + Serialize> fmt::Dis
         let mpcs_opcode_commitment = self
             .opcode_proofs
             .iter()
-            .map(|(circuit_name, (_, proof))| {
```

Comment:
These change make less readibility since now we lost circuit_name and instead only circuit index. But it's ok we fix it later, for now mpcs still be the major bottleneck of proof size